benchmarks/graphics_pipeline/main.cpp

```diff
@@ -663,11 +704,22 @@ void ProjApp::LoadNodes(
     }
 }
 
+void ProjApp::SetupCamera()
+{
+    UpdateCamera();
```

Comment:
Why is this called? Aren't the constructor default values enough?

Reply:
Actually we don't need this call. Updated.